cli/script/command-executor.ts

```diff
@@ -896,11 +896,14 @@ function getReactNativeProjectAppVersion(command: cli.IReleaseReactCommand, proj
             throw new Error(`The "CFBundleShortVersionString" key doesn't exist within the "${resolvedPlistFile}" file.`);
         }
     } else if (command.platform === "android") {
-        const buildGradlePath: string = path.join("android", "app", "build.gradle");
+        let buildGradlePath: string = path.join("android", "app", "build.gradle");
```

Comment:
It would be a plus point if we could also address this requirement that @lostintangent suggested in https://github.com/Microsoft/code-push/issues/266:

> NOTE: Since build.gradle is a required file name, it may be nice to allow this new parameter to omit it, so that you could either specify the full path to the file, or simply the directory that contains it.